serving/docker/partition/partition.py

```diff
@@ -99,20 +99,20 @@ def set_environmental_vars(self):
         if os.environ.get("PYTHONPATH"):
             python_path.append(os.environ.get("PYTHONPATH"))
         python_path.append(PYTHON_CACHE_DIR)
-        if 'model_dir' in self.properties:
-            python_path.append(self.properties['model_dir'])
+        if 'option.model_dir' in self.properties:
```

Comment:
1. Not all perperties has `option.` prefix. 
2. We don't think we are using `model_dir` in the `serving.perperties`, model_dir is passed to `model.py` ad current directory

Reply:
Alright, removed option for model_dir